code/modules/events/fake_virus.dm

```diff
@@ -0,0 +1,25 @@
+/datum/event/fake_virus/start()
+	var/list/fake_virus_victims = list()
+	for(var/mob/living/carbon/human/victim in shuffle(GLOB.player_list))
+		if(victim.stat == DEAD || victim.InCritical() || victim.mind.assigned_role == victim.mind.special_role)
+			continue
+		fake_virus_victims += victim
+
+	//first we do hard status effect victims
+	var/defacto_min = min(3, LAZYLEN(fake_virus_victims))
+	if(defacto_min) // event will hit 1-3 people by default, but will do 1-2 or just 1 if only those many candidates are available
+		for(var/i in 1 to rand(1, defacto_min))
+			var/mob/living/carbon/human/hypochondriac = pick(fake_virus_victims)
+			hypochondriac.apply_status_effect(STATUS_EFFECT_FAKE_VIRUS)
```

Comment:
Can you add logs here for admins? `hypochondriac.create_log` and then with a proper type. MISC maybe?